docker-compose.yml

```diff
@@ -22,15 +22,12 @@ services:
       args:
         GOLANG_VERSION: 1.13
     command: ["./scripts/test_unit.sh"]
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock # test this on windows, it might be different
```

Comment:
@arschles do you have a Windows machine?  Could you ensure that the `testunit` service works on Windows via the make command to run unit tests?  If it fails, could you paste the error here?

Reply:
I believe it is `make test-unit-docker`

Reply:
I can try, but let's try to get testcontainers-less appveyor tests back 😄